.github/workflows/src/context.js

```diff
@@ -210,6 +210,36 @@ export async function extractInputs(github, context, core) {
       issue_number: issue_number,
       run_id: payload.workflow_run.id,
     };
+  } else if (context.eventName === "check_run") {
+    let checkRun = context.payload.check_run;
+
+    // Extract the ADO build ID and project URL from the check run details URL
```

Comment:
What if the check_run didn't come from ADO?  I think we should handle this gracefully rather than fail.

Reply:
At this point, we only have the check_run from ADO. We could enhance it when there are other run types.